docs/src/components/DocsBehaviorRoot.tsx

```diff
@@ -25,46 +26,65 @@ class DocsBehaviorRoot extends React.Component<any, any> {
       transition: 'box-shadow 200ms, background 200ms',
       background: '#fff',
       boxShadow: '0 1px 2px #ccc',
-      margin: '10px',
+      margin: '1em 1em 1em 2em',
     }
 
     const commentBox: React.CSSProperties = {
       padding: 5,
     }
+    const topRowStyle = {
```

Comment:
it would be better to inline this style as well for now, for the sake of consistency